telco-ran/configuration/argocd/PowerConfiguration.md

```diff
@@ -70,15 +70,15 @@ The tuned performance-patch can be used to confine the maximum cpu frequency. Th
           highPowerConsumption: false
           perPodPowerManagement: true
         ...
-    - fileName: node-tuning-operator/TunedPerformancePatch.yaml
+    - fileName: node-tuning-operator/TunedPowerCustom.yaml
       policyName: "config-policy"
       spec:
         profile:
-          - name: performance-patch
+          - name: power-saving-performance-patch
             data: |
               ...
-              [sysfs]
-              /sys/devices/system/cpu/intel_pstate/max_perf_pct=72
+              [variables]
+              max_percent=72
```

Comment:
@dav1x - this is what I'm referring to - I see the max_percent variable set in the example, but no "governor"?